src/renderer/src/components/Home.jsx

```diff
@@ -164,19 +164,41 @@ function Home() {
         });
       } else if (property in bcd.css.properties) {
         userSelections.forEach(selection => {
-          const browserName = convertBrowserName(
-            browsers[selection.browser].stat,
-          );
-          const stat =
-            bcd.css.properties[property].__compat.support[browserName];
-          const isCompatible = Array.isArray(stat)
-            ? parseInt(stat[0].version_added) <= selection.version
-            : parseInt(stat.version_added) <= selection.version;
+          if (
+            typeof selection.version === "string" &&
+            selection.version.includes("-")
+          ) {
+            const versionRangeToversion = convertToNumberWithOneDecimal(
```

Comment:
```suggestion
            const versionRangeToVersion = convertToNumberWithOneDecimal(
```

Camel Case로 사용하려면 이렇게 바꿔야 할 듯 합니다.

Reply:
수정하겠습니다!